index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Template 1-mobile-main</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class="logo">
+        <a href="#">My logo</a>
+        <a><img src="./icons/menu.png" alt="menu"></a>
+    </nav>
+    <div class="home-container">
+            
+            <section class="main-info">
+                <div class="introduction">
+                       <h1>I’m Moriasi glad to see you!</h1>
+                </div>
+                <div class="more-info">
+                        <p>I’m a software developer! I can help you build a product , feature or website
+                         Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                        </p>
+                </div>    
+            </section>      
+
+            <section class="connect">
+               <h2><a href="#">Let's connect</a></h2>
```

Comment:
- [ ] Your portfolio is looking great! However, as the previous reviewer said, it would be great to add some vertical space between the paragraph and "let's connect":

|Your design|Template|
|---|---|
|![Captura de pantalla 2022-05-27 a la(s) 3 41 41 p  m](https://user-images.githubusercontent.com/83717285/170787655-d522e23b-ed57-4c62-ab9c-ac03abbadff8.png)|![Captura de pantalla 2022-05-27 a la(s) 3 43 26 p  m](https://user-images.githubusercontent.com/83717285/170787678-a9ce4d4a-bd5f-4ed6-ac8c-bbac0a96794c.png)|